lib/index.js

```diff
@@ -31,27 +34,30 @@ require('./config');
 const api = {servers: {}};
 module.exports = api;
 
+bedrock.events.on('bedrock.ready', async () => {
```

Comment:
I suspect this needs to be on `bedrock.started`, otherwise we can still hit the race condition. For example, bedrock-express only attaches its handlers on `bedrock.ready` as well, and it executes *after* this, which means there's a window during which a worker could receive a request and be unable to service it.